### PR TITLE
ZENKO-3942 - Add test on redis alerts

### PR DESCRIPTION
**Pull request by lavigne958**
add tests on redis alerts

### CODE REVIEW

File: monitoring/redis/alerts.test.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+evaluation_interval: 1m
+
+rule_files:
+  - alerts.rendered.yaml
+
+tests:
+  - interval: 30s
+    input_series:
+      - series: up{namespace="zenko", service="artesca-data-base-cache-metrics"}
+        values: 1 1 0+0x2
```
Comment (jbertran) [inline monitoring/redis/alerts.test.yaml]: You may as well be explicit here :grin:

**Review by jbertran — APPROVED** [review]

File: monitoring/redis/alerts.test.yaml (hunk 1 above)
Comment (lavigne958) [reply monitoring/redis/alerts.test.yaml]: done

**Review by francoisferrand — APPROVED** [review]

**Comment by francoisferrand** [comment]
I think this should be targeted at 2.2 branch, right?

**Comment by lavigne958** [comment]
rebased my branch to it can target dev/2.2

**Comment by lavigne958** [comment]
ping bert-e

**Pull request merged**